crates/ruff_python_parser/tests/snapshots/invalid_syntax@expressions__named__invalid_target.py.snap

```diff
@@ -198,34 +198,7 @@ Module(
 3 | (x.y := 1)
 4 | (x[y] := 1)
 5 | (*x := 1)
-  |  ^^ Syntax Error: Starred expression cannot be used here
-6 | ([x, y] := [1, 2])
-  |
-
-
-  |
-3 | (x.y := 1)
-4 | (x[y] := 1)
-5 | (*x := 1)
-  |     ^^ Syntax Error: Expected ')', found ':='
-6 | ([x, y] := [1, 2])
-  |
-
-
-  |
-3 | (x.y := 1)
-4 | (x[y] := 1)
-5 | (*x := 1)
-  |         ^ Syntax Error: Expected a statement
-6 | ([x, y] := [1, 2])
-  |
-
-
-  |
-3 | (x.y := 1)
-4 | (x[y] := 1)
-5 | (*x := 1)
-  |          ^ Syntax Error: Expected a statement
+  |  ^^ Syntax Error: Assignment expression target must be an identifier
```

Comment:
> For recovery, I assume the recovery is unchanged or did you change the recover?

The recovery for `star_named_expression` got better as a side effect. Why better? Because a named expression is actually allowed here, but it's the target expression which is invalid.